src/Aspire.Hosting.Azure.WebPubSub/Aspire.Hosting.Azure.WebPubSub.csproj

```diff
@@ -6,6 +6,8 @@
     <PackageTags>aspire hosting azure webpubsub pubsub websocket</PackageTags>
     <Description>Azure WebPubSub resource types for .NET Aspire.</Description>
     <PackageIconFullPath>$(SharedDir)AzureWebPubSub_256x.png</PackageIconFullPath>
+    <!-- Disable package validation as this package hasn't shipped yet. -->
+    <EnablePackageValidation>false</EnablePackageValidation>
```

Comment:
How do we track enabling this once these packages do ship?

Reply:
Part of the branding activities for each time we ship new packages will need to be A) to update the baseline package version to point to the package versions that we just released and B) to enable validation on any of the packages that just shipped for the first time.

So essentially just like we have to create a GH release, do branding bumps to use a new minor/patch version, and all those other activities, now we will also want to make sure we do this. I'll document this process and add the checklist of branding things to do per release into our docs in the repo.